users/users_schemas.md

```diff
@@ -0,0 +1,59 @@
+# Process schemas
+
+[User List Item](#user-list-item)  
+[User](#user)  
+[Permission](#permission)  
+[Permission Role Type](#permission-role-type)
+
+## User List Item
+
+```typescript
+type UserListItem = {
+    id: string
+  ; name: string
+}
+```
+
+## User
+
+```typescript
+type User = {
+    id: string
+  ; firstName: string
+  ; lastName: string
+  ; loginName: string
+  ; email: string
+  ; accountActive: boolean
+  ; city: string
+  ; country: string
+  ; dateOfBirth: string
+  ; lastSignInDate: string
+  ; phoneNumberLand: string
+  ; phoneNumberMobile: string
+  ; timeZoneHasDaylightSavings: boolean
+  ; timezone: string
+  ; userRole: string
+}
+```
+## Permission
```

Comment:
It probably makes sense to separate permission schemas in a separate file to avoid the duplication here and in the user group schemas file.

Reply:
I have put permission schemas in separate files